packages/ti_domaintools/data_stream/nod_feed/fields/fields.yml

```
@@ -0,0 +1,24 @@
+- name: domaintools
+  type: group
+  description: >
+    Fields for DomainTools indicators
+  fields:
+    - name: domain
+      type: keyword
+      description: >
+        The Domain. Apex-level domains (e.g. example.com but not www.example.com) that we observe for the first time, and have not observed previously with our global DNS sensor network.
+    - name: feed
+      type: keyword
+      description: >
+        The feed.
+    - name: timestamp
+      type: keyword
```

```suggestion
      type: date
```

Use a `date` processor to convert to `date`. Example: https://github.com/elastic/integrations/blob/main/packages/mimecast/data_stream/siem_logs/elasticsearch/ingest_pipeline/v2_pipeline.yml#L4-L10

---

Reference: https://www.elastic.co/guide/en/elasticsearch/reference/current/date-processor.html

---

Looking back into our docs, looks like the value we return here from our API is ISO-8601 UTC formatted timestamp. I believe that should work with the `date` type without need for a processor? Just the type switch from `keyword` to `date` should work here? lmk if I misunderstood here, thank you!

---

> Just the type switch from keyword to date should work here?

Yes, that should also work. But I suggest you add explicit `date` processor because if there is any change to API date format, then instead of dropping document (due to mapping conflict), there will be an error logged in `error.message`